sycl/include/CL/sycl/detail/builtins.hpp

```diff
@@ -32,9 +47,9 @@
   inline ALWAYS_INLINE R __invoke_##call(T1 t1) __NOEXC {                      \
     using Ret = cl::sycl::detail::ConvertToOpenCLType_t<R>;                    \
     using Arg1 = cl::sycl::detail::ConvertToOpenCLType_t<T1>;                  \
-    extern Ret PPCAT(prefix, call)(Arg1);                                      \
+    EXTERN_IT1(Ret, prefix, call, Arg1);                                       \
```

Comment:
You effectively added declarations of these `__spirv_*` built-ins to `spirv_ops.hpp`, why do we still need them here?

Reply:
These will expand to extern... lines only for host code.